packages/sanity/src/cli.ts

```diff
@@ -0,0 +1,23 @@
+
+import glob from 'glob';
+import { Command } from 'commander';
+import { hydrationTest } from './';
+import ssrTest from './ssr-test/mocha-wrapper';
+import { cliInit, defaultMetaGlob, getWebpackConfigPath } from '@ui-autotools/utils';
+
+cliInit();
+const program = new Command();
+const projectPath = process.cwd();
+const webpackConfigPath = getWebpackConfigPath(projectPath);
+
+program
+    .description('run sanity checks on all components with a metadata description')
+    .option('-f, --files [pattern]', 'Grep file')
+    .action((options) => {
+        const metaGlob: string = options.files || defaultMetaGlob;
+        glob.sync(metaGlob, { absolute: true, cwd: projectPath }).forEach(require);
```

Comment:
We have an `importMetaFiles()` function in the `scripts` package, use it here or add a TODO comment.